R/measures.R

```diff
@@ -1,9 +1,15 @@
 #' Get weather data from one station
 #'
+#' @importFrom rlang `%||%`
```

Comment:
Comparing [here](https://github.com/ropensci/riem/blob/733287a11b90c79c4b6baea91290ffca977c8031/R/utils.R#L4) and [here](https://github.com/JElchison/riem/blob/b3c5dd467b1aac70f49c25db2da5355aa4bbdb40/NAMESPACE#L9), should these backticks be removed?  I'm uncertain about the nuance of backticks vs. double-quotes.

Reply:
Oops, indeed.